paper/user_study.tex

```diff
@@ -0,0 +1,66 @@
+Our user study is mainly based on the qualitative research.
+We conduct several interviews with the professors and students as potential customers.
+We collect feedbacks from the customers, improve user experiences and fix bugs according to the feedback.
+We also use the survey to get interviewees' opinions during the user study.
+
+\subsection{Methodology}\label{subsec:methodology}
+
+\paragraph{Potential Users}
+We look for potential users to conduct the interviews.
+There are two basic standards we follow to find a potential user:
+\begin{itemize}
+    \item Professors and students who are interested in our application.
+    \item People who frequently use \cfgfiles and schema languages
+\end{itemize}
+Because each interviewee has different prior knowledge of JSON and JSON schema, we improve our interview questions and the example data also after the interview.
+So that our interview is more compatible for users.
+
+\paragraph{Interview Questions}
+We prepare our interview questions based on a mockup JSON schema and \cfgfiles.
+It is about a simple and understandable autonomous vehicle schema.
```

Comment:
It's not about a autonomous vehicle schema. It's about a (made-up) simulation software that simulates a self-driving car.